docs/deployment/migration.md

```diff
@@ -24,6 +24,8 @@ straight to [Getting Started](getting-started.md).
 ## Migration Steps
 
 ### 1. Configure System Extensions
+- (Optional) Add a Team ID Rule for North Pole Security's Team ID (ZMCG7MLDV9) to Google Santa
```

Comment:
My only concern with this is that being an optional step, how does an admin following these steps decide whether they should or should not do it? Could we add another bullet outlining the reason to do it?

Reply:
Makes sense. How about a sentence like the following:

`(Optional) Add a Team ID Rule for North Pole Security's Team ID (ZMCG7MLDV9) to Google Santa. This is to guarantee that complex MDM setups don't allow Google Santa to block NPS Santa`

Reply:
Works for me